tests/Unit/Models/GroupTest.php

```diff
@@ -15,78 +15,75 @@ class GroupTest extends TestCase
     public function groupsHaveMembers()
     {
         $group = Group::factory()->create();
-        $user1 = User::factory()->create();
-        $user2 = User::factory()->create();
-        User::factory()->count(4)->create();
+
+        $firstUserInGroup  = User::factory()->create();
+        $secondUserInGroup = User::factory()->create();
+
         Member::factory()->create([
             'group_id' => $group->id,
-            'user_id'  => $user1->id,
+            'user_id'  => $firstUserInGroup->id,
         ]);
+
         Member::factory()->create([
             'group_id' => $group->id,
-            'user_id'  => $user2->id,
+            'user_id'  => $secondUserInGroup->id,
         ]);
 
-        $this->assertEquals(2, $group->members->count());
+        $this->assertInstanceOf(
+            User::class,
+            $group->members->first(),
+        );
+
         $this->assertEquals(
-            [$user1->id, $user2->id],
+            [$firstUserInGroup->id, $secondUserInGroup->id],
             $group->members->pluck('id')->sort()->toArray(),
         );
     }
 
     /** @test */
-    public function groupsDontHaveMembersWhenNoneAdded()
+    public function groupsCanBeScopedToUsers()
     {
-        $group = Group::factory()->create();
+        $groups = Group::factory()->count(2)->create();
 
-        $this->assertEquals(0, $group->members->count());
-    }
+        $userInTwoGroups  = User::factory()->create();
+        $userInZeroGroups = User::factory()->create();
 
-    /** @test */
-    public function groupsCanBeScopedToUsers()
-    {
-        $groups = Group::factory()->count(5)->create();
-        $user1  = User::factory()->create();
-        $user2  = User::factory()->create();
-        User::factory()->count(5)->create();
         $groups[0]->memberships()->create([
-            'user_id' => $user1->id,
+            'user_id' => $userInTwoGroups->id,
         ]);
         $groups[1]->memberships()->create([
-            'user_id' => $user1->id,
+            'user_id' => $userInTwoGroups->id,
         ]);
 
-        $groupsForUser1 = Group::query()
-                            ->hasMember($user1)
-                            ->get();
+        $groupsForUserInTwoGroups = Group::query()
+                                        ->hasMember($userInTwoGroups)
```

Comment:
Why are these lines right-tabbed into the next county?...

```
Thing::stuff()
    ->andAlso()
    ->someOtherStuff()
    ->right_here;
```

Reply:
So that I can see them next to the model name. I can move them to 1 indent, though.

Reply:
Fixed for the tests, and I can fix it if I find it in other places in the future.